ansible/roles/automation/setup/tasks/enable_nested_virt.yml

```diff
@@ -0,0 +1,69 @@
+# enable nested virtualization so we can build templates in VM
```

Comment:
This code seems to be a duplicate of ansible/roles/runner/tasks/enable_nested_virt.yml. I'd rather avoid code duplication if it's possible.

Reply:
you right, this is duplication:

diff enable_nested_virt.yml ../../../runner/tasks/enable_nested_virt.yml 
1,2d0
< # enable nested virtualization so we can build templates in VM
< 

I fix this.